format-specs/geoparquet.md

```diff
@@ -80,10 +80,10 @@ which itself implements the model of
 Apart from the difference of encodings, the semantics are intended to match
 WKT2:2019, and a CRS in one encoding can generally be represented in the other.
 
-If CRS is not provided, all coordinates in the geometry must use longitude, latitude
-based on the WGS84 datum to store their data.
+If the `crs` field is not provided, the default CRS is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84),
+which means that all coordinates in the geometries must use longitude, latitude based on the WGS84 datum.
 
-If an implementation is CRS-aware and needs a CRS representation of the data it should assume a default value is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84), which is equivalent to the well-known [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from latitude-longitude to longitude-latitude.
+[OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84) is equivalent to the well-known [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from latitude-longitude to longitude-latitude.
```

Comment:
No issue. That twisted language came from some idea that there is data that people put in lat/long that is not exactly crs84, due to some sub-parameters of the WKT, so this was to somehow communicate that it doesn't need to be exact, or something... I forget the exact thing that lead to this. But since I can barely remember what we were trying to achieve I'm in support of clearer language.

Reply:
Could you have a look again? I tried to squeeze out the "can assume" while trying to keep your intention.